pgsql-flexible-server.tf

```diff
@@ -2,10 +2,18 @@ locals {
   default_name = var.component != "" ? "${var.product}-${var.component}" : var.product
   name         = var.name != "" ? var.name : local.default_name
   server_name  = "${local.name}-${var.env}"
+  vnet_rg_name = var.project == "sds" ? "ss-${var.env}-network-rg" : "${var.project}-${var.env}-network-rg"
+  vnet_name    = var.project == "sds" ? "ss-${var.env}-vnet" : "${var.project}-${var.env}-vnet"
 
   private_dns_zone_id = "/subscriptions/1baf5470-1c3e-40d3-a6f7-74bfbce4b348/resourceGroups/core-infra-intsvc-rg/providers/Microsoft.Network/privateDnsZones/private.postgres.database.azure.com"
 }
 
+data "azurerm_subnet" "pg_subnet" {
+  name                 = "postgresql"
```

Comment:
this needs a provider resource as in CFT Jenkins runs the terraform against a different subscription.
(and a different vnet core-infra-vnet-${var.env)) this repo for the network there I think https://github.com/hmcts/cnp-core-infrastructure/blob/master/main.tf which will need a new `postgresql` subnet (sorry @TomKimber I forgot about this yesterday)

see how it's done in https://github.com/hmcts/terraform-module-servicebus-namespace/blob/master/private-endpoint.tf#L10

(there used to be a `data` provider in this module that I removed because it was unused, but you could restore that.

(add it in the `configuration_aliases` block https://github.com/hmcts/terraform-module-servicebus-namespace/blob/master/providers.tf#L6)

Reply:
Updated to use core-infra-vnet CFT resources and include the configuration alias requested.

Reply:
Removed.